include/deal.II/lac/relaxation_block.h

```diff
@@ -147,6 +157,16 @@ class RelaxationBlock :
      * </ol>
      */
     std::vector<std::vector<unsigned int> > order;
+
+    /**
+     * Temporary ghost vector that is used in the Relaxation method when
+     * performing parallel MPI computations. The user is required to have this
+     * point to an initialized vector that contains at least the locally
+     * active level DoFs. Unused when VectorType is a serial vector type
```

Comment:
That's correct in most practical cases, but what you really need are all DoFs that are listed in the `block_list`, right?